.github/workflows/scripts/mailchimp/index.js

```diff
@@ -27,8 +27,14 @@ module.exports = async (link, title) => {
             type: 'regular',
             recipients: {
                 list_id: '6e3e437abe',
-                segments_opts: {
-                    saved_segment_id: 'tsc-voting-email'
+                segment_opts: {
```

Comment:
```suggestion
                segments_opts: {
```

are you sure it is correct? 😆

Reply:
Are you? 😂